### PR TITLE
Adding my_gpcal_package

### PR DESCRIPTION
**Pull request by jhparkastro**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jhparkastro** [comment]
Please add my package to Conda-forge.

File: recipes/gpcal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ca87d0418e51cb9b7334b2d48a6102c2730eeb687e4a6a5f85585e7ea98cda07
+
+build:
+  number: 0
```
Comment (scopatz) [inline recipes/gpcal/meta.yaml:13]: This can be noarch: python?

File: recipes/gpcal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - astropy
+    - matplotlib
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy
```
Comment (scopatz) [inline recipes/gpcal/meta.yaml:24]: Are all of these needed to build the package?

File: recipes/gpcal/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - astropy
+    - matplotlib
```
Comment (scopatz) [inline recipes/gpcal/meta.yaml:27]: This should be `matplotlib-base`

File: recipes/gpcal/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+
+about:
+  home: "https://github.com/jhparkastro/gpcal"
+  license: "GNU General Public v2 or later (GPLv2+)"
+  license_family: GPL2
+  license_file: 
+  summary: "A generalized instrumental polarization calibration pipeline for VLBI data"
+  doc_url: 
+  dev_url:
```
Comment (scopatz) [inline recipes/gpcal/meta.yaml:41]: Please remove empty entries

File: recipes/gpcal/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  license: "GNU General Public v2 or later (GPLv2+)"
+  license_family: GPL2
+  license_file: 
+  summary: "A generalized instrumental polarization calibration pipeline for VLBI data"
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (scopatz) [inline recipes/gpcal/meta.yaml:45]: Please add your github name

**Comment by jhparkastro** [comment]
It seems that my recipe still cannot pass the tests. I'm afraid this is probably because my package uses another package called parseltongue. It imports something that cannot be installed by pip so I could not put it on the requirements of my meta.yaml file. But that package can be easily installed by conda like "conda install -c kettenis parseltongue". Do you have any idea how to incorporate this requirement into my meta.yaml file? Thank you very much for your support and consideration.

**Comment by scopatz** [comment]
In this case, you would need to have `parseltongue` on codna-forge.  Please put in another PR for that package

**Comment by jhparkastro** [comment]
Thanks for your comments. But I'm not an author of that package and I'm afraid whether I can upload the package to conda-forge (and how?)......

**Comment by scopatz** [comment]
@jhparkastro  - if it is this project, it is MIT licensed and we are free to package it. You would package it like any other package on conda forge: https://pypi.org/project/parseltongue/

**Comment by jhparkastro** [comment]
@scopatz I think that is a different package with the same name. The one that I need to use is here: http://old.jive.nl/jivewiki/doku.php?id=parseltongue:parseltongue. It seems it is not in PyPi.

**Comment by scopatz** [comment]
That code (whose tarball is here: http://www.jive.nl/parseltongue/releases/parseltongue-2.3.tar.gz) is GPL v2, so you can build and distribute that on conda-forge too

**Comment by jhparkastro** [comment]
@scopatz Thank you very much for your comments, Anthony. It seems that it is not a simple python-only package and I have no idea how to construct meta.yaml file for this package. Could you recommend me any guidance document if there is any? I'm so sorry for bothering you.

**Comment by scopatz** [comment]
There are a lot of examples in conda-forge in our feedstocks on how to package C/C++.  But here are some examples as well: https://docs.conda.io/projects/conda-build/en/latest/user-guide/recipes/sample-recipes.html

**Comment by scopatz** [comment]
Basically you need to write a build.sh script

**Comment by jhparkastro** [comment]
Thank you very much! Those should be very helpful for me. Let me try.

**Pull request closed without merging**